modules/mobile/app/controllers/mobile/v0/authorized_services_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    class AuthorizedServicesController < ApplicationController
+      def show
```

Comment:
Could this be considered an index? I can't think of a scenerio where we'd have a index and show for this controller so the distinction probably doesn't matter.

Reply:
We're like 2:1 index:show so we're inconsistent enough that I don't really care too much, but index is the majority so I'll leave it up to @Tonksthebear to decide.

Reply:
Happy to switch to index. I originally wrote it as index but then backed out because I thought it could be interpreted as "authorized services" is a `has_one` relationship to the user, which in normal resourceful routing would make this a `show`. But I also could see it as a `has_many` if each service is considered a resource. In any case, switching is easy and I'm in the middle on it